schema/settings.json

```diff
@@ -14,6 +14,12 @@
       "title": "Highlight",
       "description": "Highlight the execution time on completion",
       "default": true
+    },
+    "display": {
```

Comment:
Naming is hard. I'm waffling here, but maybe we should call this `positioning`? `display` is overloaded due to CSS, as is `position`.

Reply:
I don't believe this clashes with css-names. But for user friendliness `positioning` is indeed better.

Reply:
I don't believe this clashes with css-names. But for user friendliness `positioning` is indeed better.